function-server/src/test/java/io/dispatchframework/javabaseimage/executor/SimpleFunctionExecutorTests.java

```diff
@@ -102,16 +100,21 @@ public void test_execute_mismatchedPayload() {
         SimpleFunctionExecutor principal = new SimpleFunctionExecutor(helloFunction);
 
         String message = "{\"context\": null, \"payload\": \"invalid\"}";
-        String actual = principal.execute(message);
-        assertTrue(actual.contains(ErrorType.INPUT_ERROR.toString()));
+        try {
+            principal.execute(message);
+        } catch (DispatchException e) {
+            assertTrue(e.getError().contains(ErrorType.INPUT_ERROR.toString()));
+        }
```

Comment:
Might be better to use `assertThrows` to make sure an exception is thrown, something like:
```
DispatchException e = assertThrows(DispatchException.class, principle.execute(message))
assertTrue(e.getError().contains(ErrorType.INPUT_ERROR.toString()));
```